### ISSUE TITLE
debian bullseye repo changed

### ISSUE THREAD

**Issue by plinkable** [fluent/fluent-bit#6732]
The debian repo for bullseye has changed and is throwing the following error:

``` 
E: Repository 'https://packages.fluentbit.io/debian/bullseye bullseye InRelease' changed its 'Origin' value from 'bullseye bullseye' to '. bullseye'
E: Repository 'https://packages.fluentbit.io/debian/bullseye bullseye InRelease' changed its 'Label' value from 'bullseye bullseye' to '. bullseye'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
```

It's probably best to change this back in the short term, and perhaps determine if a cleaner value could be provided for future codenames.

**Comment by patrick-stephens** [comment]
Looking into this, the underlying issue seems to be a slightly different configuration on the old server just for the Bullseye repo: all other repos follow this format then and now. As the automated process is identical for all repos this means the new server now aligns with it.

As it is blocking updates, we should probably align only the Bullseye repo configuration with what the old server did which involves setting the `Origin` and `Label` fields via aptly here: https://github.com/fluent/fluent-bit/blob/d90e07424838bc1b289bee6bbc40f99929ab1d57/packaging/update-repos.sh#L114

See https://www.aptly.info/doc/aptly/publish/repo/

**Comment by patrick-stephens** [comment]
This should be resolved now using the changes in #6733 :
```shell
$ curl https://packages.fluentbit.io/debian/bullseye/dists/bullseye/Release
Origin: bullseye bullseye
Label: bullseye bullseye
Suite: bullseye
Codename: bullseye
Date: Wed, 25 Jan 2023 14:50:29 UTC
Architectures: amd64 arm64
Components: main
Description: Generated by aptly
```
The other repos should be unaffected:
```shell
$ curl https://packages.fluentbit.io/debian/buster/dists/buster/Release
Origin: . buster
Label: . buster
Suite: buster
Codename: buster
Date: Mon, 16 Jan 2023 16:37:39 UTC
Architectures: amd64 arm64
Components: main
Description: Generated by aptly
```
The only thing to be aware of is any caching you may have of the metadata needs refreshing.
